src/System.ServiceModel.Primitives/ref/project.json

```diff
@@ -13,6 +13,8 @@
     "System.Reflection": "4.0.0",
     "System.Net.Primitives": "4.0.0",
     "System.Security.Principal": "4.0.0",
+    "System.Security.Cryptography.X509Certificates" : "4.0.0-beta-*",
```

Comment:
You are adding a -beta-* dependency which is an implementation version to a contract. Shouldn't this be 4.0.0? Otherwise we will need to bump our contract version when we move out of beta simply to update this reference to remove the -beta-*..